docs/source/guides/new_user_guide.md

```diff
@@ -53,7 +53,30 @@ If you have 3 credits that expire on May 1, after depositing 10 ADA, you will ha
 
 **Price List:**
 
+Metadata:
+
 - SubmitMetadata: 10 credits
 - ListTransactions: 1 credit
+
+Store and hash:
+
 - AuthenticatedEmail: 15 credits
-- HashAndStore: 1 credit
\ No newline at end of file
+- HashAndStore: 1 credit
+
+Native assets:
+
+- createPolicy: 5 credits
+- deletePolicy: 0 credits
+- getPolicy: 0 credits
+- getPolicies: 0 credits
+- createAsset: 5 credits
+- deleteAsset: 0 credits
+- listAsset: 0 credits
+- listAssets: 0 credits
+- mint: 25 credits
+- fund 1 ADA: 25 credits
+- transfer: 25 credits
+- burn: 25 credits
+- importPolicy: 0 credits
+- airdrop: 0 credits
```

Comment:
Is this correct? If so, aren't we a little bit too generous?

Reply:
As far as I know, the list and prices wee approved by Ilan and added by Adam.

Reply:
actually it is not up to date:
```
paywall {
    native-assets {
      createPolicyCost = 5
      deletePolicyCost = 5
      getPolicyCost = 1
      getPoliciesCost = 1
      createAssetCost = 5
      deleteAssetCost = 5
      getAssetCost = 1
      getAssetsCost = 1
      mintCost = 150
      fund1AdaCost = 25
      transferCost = 25
      burnCost = 25
      importPolicyCost = 5
      airDropCostPerNft = 25
      airDropStatusCost = 1
      multisig {
        createPolicyCost = 5
        importPolicyCost = 5
        createMintTxCost = 150
        createTransferTxCost = 1
        createBurnTxCost = 1
        addWitnessCost = 1
        sendTxCost = 25
        getPolicyCost = 1
        listPoliciesCost = 1
        getTxCost = 1
        listTxsCost = 1
        listWitnessesCost = 1
      }
   }
}```

thank you for @mpskowron for pointing this out